spec/spec_helper.rb

```diff
@@ -27,17 +27,17 @@
 require "rspec/longrun"
 #require "narray/narray"
 
-require "../lib/nmatrix"
-require "../lib/nmatrix/rspec"
+require "./lib/nmatrix"
+require "./lib/nmatrix/rspec"
```

Comment:
I can see the change in commit history. 
One step is missing.
I am using git blame and can't figure out, how to revert this changes from the history as I am unable to find the commit that changed the path initially.